src/components/content/catalog/services/update/UpdateService.tsx

```diff
@@ -199,6 +214,7 @@ function UpdateService({
                         <UpdateResult
                             ocl={ocl.current}
                             serviceRegistrationStatus={serviceRegistrationStatus.current}
+                            isAvailableInCatalog={isAvailableInCatalog.current}
```

Comment:
for update and publish, we must refer to the 'ServiceTemplateRequestInfo' object's requestSubmittedForReview field.